pkg/kube/wrappers/ingress.go

```diff
@@ -113,6 +113,13 @@ func (iw *IngressWrapper) supportsTLS() bool {
 	return false
 }
 
+func (iw *IngressWrapper) getTLSHosts() []string {
+	if iw.supportsTLS() {
+		return iw.ingress.Spec.TLS[0].Hosts
+	}
+	return []string{}
+}
+
```

Comment:
any reason for this function? you could simply check for `len(iw.ingress.Spec.TLS[0].Hosts)` on the check at line 102.
The `&&` will ensure that the second condition is only checked if the first is true.

Reply:
The only reason I had added this function was as a future-proofing measure. If we end up needing the the TLSHosts slice for some other task then it can come in handy. 

If it seems early for that, i'll remove it.

Reply:
Yeah, the function can be added when required in the future

Reply:
done